UI/window-basic-preview.cpp

```diff
@@ -2357,7 +2357,10 @@ static obs_source_t *CreateLabel(float pixelRatio)
 	const char *text_source_id = "text_ft2_source";
 #endif
 
-	return obs_source_create_private(text_source_id, NULL, settings);
+	struct dstr name;
+	dstr_init(&name);
+	dstr_printf(&name, "Preview spacing label %d", i);
+	return obs_source_create_private(text_source_id, name.array, settings);
```

Comment:
This resulted in a memory leak.

Reply:
Oh right, obviously, I forgot the dstr_free part. I’ll fix it tomorrow.

Reply:
Fixed